src/pages/footer.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="tr">
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Sayfa Sonu Footer</title>
+  <link rel="stylesheet" href="/public/assets/css/footer.css" />
+  <style>
+
+  </style>
+</head>
+<body>
+
+  <main>
+    <h1>Merhaba Dünya</h1>
+    <p>Bu bir örnek içeriktir. Sayfa içeriği burada yer alır.</p>
+  </main>
+
+  <footer class="footer">
+    <div class="footer-container">
+      <div class="footer-section">
+        <h4>Hakkımızda</h4>
+        <p>Biz yaratıcı çözümler üreten bir web geliştirme ekibiyiz.</p>
+      </div>
+      <div class="footer-section">
+        <h4>Bağlantılar</h4>
+        <ul>
+          <li><a href="#">Ana Sayfa</a></li>
+          <li><a href="#">Hizmetler</a></li>
+          <li><a href="#">İletişim</a></li>
+        </ul>
+      </div>
+      <div class="footer-section">
+        <h4>İletişim</h4>
+        <ul>
+          <li><a href="#">info@ornek.com</a></li>
+          <li><a href="#">+90 555 123 4567</a></li>
+          <li><a href="#">İstanbul, Türkiye</a></li>
+        </ul>
+      </div>
+    </div>
+    <div class="footer-bottom">
+      &copy; 2025 Tüm hakları saklıdır | Zehra Öztürk
+    </div>
```

Comment:
Eğer ki yazdığın scripteki gibi dinamik olarak bu bölümü oluşturmak istiyorsan bu şekilde yazman daha doğru olur. Çünkü şu an yazdığın kod hem kendini tekrar ediyor hem de mantıken bir karmaşa yaratıyor. Ya static olarak tarihi ve metini eklemelisin ya da dinamik olarak.

```suggestion
    <div class="footer-bottom">
      <p></p>
    </div>
```